frontend/src/components/FileUploader/FileTable.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-unknown-property */
 import React, { useState, useRef } from 'react';
```

Comment:
Is this needed?

Reply:
yeah, otherwise the linter is flagging the "bgcolor" attribute on the thead as invalid. It must not be valid react, but it compiles to valid HTML.